pkg/scheduler/framework/plugins/nodelabel/node_label.go

```diff
@@ -37,25 +39,37 @@ type Args struct {
 	Labels []string `json:"labels,omitempty"`
 	// The boolean flag that indicates whether the labels should be present or absent from the node
 	Presence bool `json:"presence,omitempty"`
+
+	// The parameters that are used to configure the priority function
+	// The label that identify node "groups"
+	PreferenceLabel string `json:"preferenceLabel,omitempty"`
```

Comment:
Do we know if the label/presence for the priority has anything to do with the labels/presence for the predicate? Or are they completely independent?

cc @ahg-g

Reply:
we are going to change our approach with regards to custom predicates/priorities, @liu-cong will have an update soon.